coral-component-multifield/src/scripts/Multifield.js

```diff
@@ -751,6 +751,9 @@ const Multifield = Decorator(class extends BaseComponent(HTMLElement) {
     // a11y
     this._handleRoleList();
 
+    // a11y Add aria-label to the add button to give context to screen reader users
+    $("[coral-multifield-add]").attr("aria-label","Add a new item to the options list");
```

Comment:
Coral-multifield is a generic component, the string doesn't seem like one. Can you change it to  more generic string
Coral has no dependency on jquery, please don't use it.